### DISCUSSION TITLE
What’s the point of the json() function?

### DISCUSSION THREAD

**Discussion by sinclairnick** [remix-run/remix#9468] [General]
I'm not sure what the point of the json() function is?

The Web Standards already provide the Response.json function which seemingly does the same thing.

Perhaps it's doing other stuff under the hood?

**Comment by sergiodxa** [comment]
When it was implemented Response.json was not a thing, and now it returns a TypedResponse to get type inference working

Also it will be deprecated with Single Fetch feature where it's not needed anymore.

**Reply by sinclairnick** [reply]
Makes sense. Thought that might have been the case.

Thank you for the insight!
